src/nspawn/nspawn.c

```diff
@@ -1395,8 +1395,15 @@ static int copy_devnodes(const char *dest) {
 
                 } else {
                         if (mknod(to, st.st_mode, st.st_rdev) < 0) {
-                                if (errno != EPERM)
-                                        return log_error_errno(errno, "mknod(%s) failed: %m", to);
+                                if (errno != EPERM) {
+                                        log_error("mknod(%s) failed: %m", to);
+
+                                        /* For now tell users that we need an empty /dev */
+                                        if (errno == EEXIST)
+                                                log_error("Failed '%s/dev/' already populated.", dest);
```

Comment:
This is gramatically incorrent: "failed" is an adjective that refers to the path, and the path itself cannot fail.

Also, _errno version should still be used. The errno gets stored in the journal as metadata, so we want it, even if not necessary for the error message.